packit/cli/utils.py

```diff
@@ -0,0 +1,34 @@
+import functools
+import logging
+import sys
+
+from packit.exceptions import PackitException
+
+logger = logging.getLogger(__name__)
+
+
+def cover_packit_exception(_func=None, *, exit_code=1):
+    """
+    Decorator for executing the function in the try-except block.
+
+    The PackitException is caught and
+    - raised in debug
+    - sys.exit(exit_code), otherwise
+
+    """
+
+    def decorator_cover(func):
+        @functools.wraps(func)
+        def covered_func(*args, **kwargs):
+            try:
+                func(*args, **kwargs)
+            except PackitException as exc:
```

Comment:
please also add KeyboardInterrupt and a generic Exception handler

https://github.com/TomasTomecek/sen/blob/master/sen/cli.py#L70

(no need to print anything for the KeyboardInterrupt)

Reply:
Fixed